src/style.css

```diff
@@ -1,3 +1,66 @@
+.logo {
+  margin-top: 6px;
+  padding: 10px 0;
+  height: 40px;
+  width: 40px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background: white;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding-left: 50px;
+}
+
+.nav {
+  padding-right: 30px;
+  margin-top: 23px;
+}
+
+.nav_list {
+  list-style-type: none;
+  display: flex;
+}
+
+.nav_link {
+  padding-right: 20px;
+  text-decoration: none;
+  color: black;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+  transition: color 0.33s;
```

Comment:
you should add transition on the base class, because now transition will work only when you hovering link, when you will remove cursor this won't work

Reply:
done